recourse_methods/mrmc_method.py

```diff
@@ -95,15 +95,15 @@ def filter_data(self, confidence_threshold: float, model: base_model.BaseModel)
         self.X = self.X[p >= confidence_threshold]
         return self
 
-    def get_unnormalized_direction(self, poi: dp.EmbeddedSeries) -> dp.EmbeddedSeries:
+    def get_unnormalized_direction(self, poi: recourse_adapter.EmbeddedSeries) -> recourse_adapter.EmbeddedSeries:
         """Returns an unnormalized recourse direction."""
         diff = (self.X - poi).to_numpy()
         dist = np.sqrt(np.power(diff, 2).sum(axis=1))
         alpha_val = self.alpha(dist)
         dir = diff.T @ alpha_val
```

Comment:
`diff` and `dist` here should be (hopefully) fairly understandable to a new reader, but `dir` again is less so (it's common shorthand for directory).